alfresco-transform-misc/alfresco-transform-misc/src/main/java/org/alfresco/transformer/transformers/StringExtractingContentTransformer.java

```diff
@@ -126,11 +126,22 @@ public void transform(final File sourceFile, final File targetFile, final String
         {
             if (charReader != null)
             {
-                try { charReader.close(); } catch (Throwable e) { logger.error(e); }
+                try {
+                    charReader.close();
+                }
+                catch (Throwable ignore)
+                {

```

Comment:
Why remove log error (even in close) ? And below ...

Reply:
Restored (using equivalent org.slf4j.Logger method) along with the original poor formatting.

Reply:
OK, thanks.